app/Http/Controllers/Auth/LoginController.php

```diff
@@ -74,6 +74,13 @@ public function showLoginForm(Request $request)
             return redirect()->intended('/');
         }
 
+        //If the environment is set to ALWAYS require SAML, go straight to the SAML route.
+        //We don't need to check other settings, as this should override those.
```

Comment:
We don’t typically use `env()` variables in Snipe-IT. We add those to the config file so we can access them as `config(‘foo.bar’)`

Reply:
That makes sense. I didn't do that for any specific reason, so I am more than willing to change that.